.changeset/39-add-no-default-export-rule.md

```diff
@@ -0,0 +1,6 @@
+---
+issue: https://github.com/pmedianetwork/eslint-config/issues/39
+type: minor
```

Comment:
Not sure if we have reached a consensus yet on the issue on how to implement this rule. But minor release with the rule set to error was definitely NOT on the list.

Reply:
Sorry, my bad.

Reply:
I added the `discussion` label to the issue.